lang/main-az.json

```diff
@@ -0,0 +1,1252 @@
+        "kickParticipant": "{{kicked}} {{kicker}} tərəfindən təpikləndi",
+        "leftOneMember": "{{name}} görüşdən çıxdı",
+        "leftThreePlusMembers": "{{name}} və bir çox başqaları iclası tərk etdi",
+        "leftTwoMembers": "{{first}} və {{ikinci}} görüşdən çıxdı",
+        "me": "Mən",
+        "moderationInEffectCSDescription": "Ekranınızı paylaşmaq istəyirsinizsə, lütfən, əlinizi qaldırın.",
+        "moderationInEffectCSTitle": "Ekran paylaşımı moderator tərəfindən bloklanıb",
+        "moderationInEffectDescription": "Danışmaq istəyirsinizsə, lütfən, əl qaldırın.",
+        "moderationInEffectTitle": "Mikrofonunuz moderator tərəfindən səsi kəsilib",
+        "moderationInEffectVideoDescription": "Kameranı işə salmaq istəyirsinizsə, lütfən, əlinizi qaldırın.",
+        "moderationInEffectVideoTitle": "Kameranınız moderator tərəfindən bloklanıb",
+        "moderationRequestFromModerator": "Aparıcı səsi açmağınızı istəyir",
+        "moderationRequestFromParticipant": "Danışmaq istəyir",
+        "moderationStartedTitle": "Moderasiya başladı",
+        "moderationStoppedTitle": "Moderasiya dayandırıldı",
+        "moderationToggleDescription": "{{participantDisplayName}} tərəfindən",
+        "moderator": "Sən artıq moderatorsan",
+        "muted": "Söhbəti susdurmağa başladınız.",
+        "mutedRemotelyDescription": "Danışmağa hazır olduğunuz zaman səsi istənilən vaxt aça bilərsiniz. Görüşdən səs-küy uzaqlaşdırmaq üçün işiniz bitdikdə səsi söndürün.",
+        "mutedRemotelyTitle": "Siz {{participantDisplayName}} tərəfindən səssizləşdirildiniz",
+        "mutedTitle": "Səssizsən!",
+        "newDeviceAction": "İstifadə et",
+        "newDeviceAudioTitle": "Yeni audio cihazı aşkarlandı",
+        "newDeviceCameraTitle": "Yeni kamera aşkarlandı",
+        "oldElectronClientDescription1": "Görünür, siz Jitsi Meet müştərisinin məlum təhlükəsizlik zəiflikləri olan köhnə versiyasından istifadə edirsiniz. Lütfən, ",
+        "oldElectronClientDescription2": "son quruluş",
+        "oldElectronClientDescription3": "indi!",
+        "participantWantsToJoin": "Görüşə qoşulmaq istəyir",
+        "participantsWantToJoin": "Görüşə qoşulmaq istəyirsiniz",
+        "passwordRemovedRemotely": "$t(lockRoomPasswordUppercase) başqa iştirakçı tərəfindən silindi",
+        "passwordSetRemotely": "$t(lockRoomPasswordUppercase) başqa iştirakçı tərəfindən təyin edilib",
+        "raiseHandAction": "Əli qaldır",
+        "raisedHand": "Danışmaq istərdim.",
+        "raisedHands": "{{participantName}} və daha {{raisedHands}} nəfər",
+        "reactionSounds": "Səsləri söndürün",
+        "reactionSoundsForAll": "Hamı üçün səsləri söndürün",
+        "screenShareNoAudio": "Pəncərə seçim ekranında audio paylaşma qutusu işarələnməyib.",
+        "screenShareNoAudioTitle": "Sistem audiosunu paylaşmaq mümkün olmadı!",
+        "selfViewTitle": "Siz həmişə parametrlərdən öz görünüşünü gizlədə bilərsiniz",
+        "kimsə": "Kimsə",
```

Comment:
While I don't speak your language, I'm feeling this may be a mistake; maybe "somebody": "Kimsa" ?

Reply:
yes